src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -117,10 +117,12 @@ static constexpr float HDG_HOLD_YAWRATE_THRESH = 0.15f;
 // [.] max manual roll/yaw normalized input from user which does not change the locked heading
 static constexpr float HDG_HOLD_MAN_INPUT_THRESH = 0.01f;
 
-// [us] time after which we abort landing if terrain estimate is not valid
+// [us] time after which we abort landing if terrain estimate is not valid. this timer start whenever the terrain altitude
+// was previously valid, and has changed to invalid.
 static constexpr hrt_abstime TERRAIN_ALT_TIMEOUT = 1_s;
 
-// [us] if no altimeter measurement is made within this timeout, the land waypoint altitude is used for terrain altitude
+// [us] within this timeout, if a distance sensor measurement not yet made, the land waypoint altitude is used for terrain
```

Comment:
We should mention in the docs at least that the slope shouldn't start above the max distance sensor range.

Reply:
yeah, agreed. at least until we get a next PR going with utilization of the driver specs for max/min range

Reply:
Right. Let's also add a comment in the param description of FW_LND_USETER.

Reply:
done in [9161270](https://github.com/PX4/PX4-Autopilot/pull/19871/commits/91612704277675ae59f91d370b34642a9d8f682b)